customer-wallet/src/store/modules/data/payment.js

```diff
@@ -69,6 +70,8 @@ export function submitPayment (data) {
       dispatch(setPaymentSuccess())
     } catch(exception) {
```

Comment:
I think we should split this into multiple try blocks so that we can know if it was a transaction error or a chlu error